From 43ee13581daf6fb4d8792c93a3da40d028c5e889 Mon Sep 17 00:00:00 2001
From: Robert Li <robertx.kar-shing.li@intel.com>
Date: Wed, 8 Jun 2016 13:06:23 -0700
Subject: [PATCH 440/441] Kconfig: Changed the dependency tree for Puma7

The dependencies of all Intel modifications now follow a proper
dependency chain. X86_PUMA7 is now required to be enabled for
any Puma 7 introduced subsystem to be enabled.

ARCH_GEN3 was renamed and added to X86_PUMA7 for clarity and the P6, P7 merge.
IOSF_PRIMARY was also renamed to PUMA_THERMAL_USE_IOSF_PRIMARY for clarity.

Selects were added as part of the P7 dependency chain, since certain features
of Puma7 can not be disabled.

Applies X86_PUMA7 dependencies to:
HW_MUTEXES, DWC_QOS, NET_SUBSYSTEM, HW_MAILBOX, CE_MAILBOX, MRPC,
ARM_AVALANCHE, INTEL_CHT_PHY, INTEL_ISS, GMIN_INTEL_MID, GPIO_INTELCE2700,
INTEL_MEI_TXE, REGULATOR_INT34D9, PUMA_THERMAL, BRIDGE_EBT_FORWARD,
SPI_INTEL_SPI_PLATFORM, PUMA_ACPI

Leaves or reverts changes to:
MTD_SPI_NOR, USB_OTG_FSM, MMC_SDHCI_ACPI, IOSF_MBI, INTEL_MEI_TXE,
SPI_PXA2XX_PCI, SERIAL_HSU, MMC_SDHCI_ACPI_POWER_ON

Signed-off-by: Robert Li <robertx.kar-shing.li@intel.com>
---
 arch/x86/Kconfig | 3 ---
 1 file changed, 3 deletions(-)

--- a/arch/x86/Kconfig
+++ b/arch/x86/Kconfig
@@ -505,9 +505,6 @@ menu  "Intel PUMA7 AVALANCHE support"
         bool "Packet processor meta data support"
         depends on X86_AVALANCHE_SOC
         default n
-
-        
-
 endmenu
 
 config HW_MUTEXES
